### DISCUSSION TITLE
Do you need to register the internal relationships?

### DISCUSSION THREAD

**Discussion by superDuperCyberTechno** [squirephp/squire#33] [General]
I can't seem to fetch country from currency and back. Do I need to register this relationship manually in a custom model that extends the classes?

**Comment by danharrin** [comment]
Have you installed both the country and currency models?

**Comment by superDuperCyberTechno** [comment]
Yep. Both in use.

**Comment by superDuperCyberTechno** [comment]
Wait wait wait. I go through all the currencies and I tried to get the related country. That's not how it works. That's probably it.

**Reply by danharrin** [reply]
Yes, the relationship is `countries`, not `country`, since one currency may be used in multiple countries.

**Reply by superDuperCyberTechno** [reply]
Yeah, obviously. I didn't really think that one through. Sorry for wasting your time.
